src/rest/connector/libs/nd/implementation.py

```diff
@@ -32,7 +32,7 @@ class Implementation(Imp):
                 credentials:
                     rest:
                         username: admin
-                        password: cisco123
+                        password: [REDACT]
```

Comment:
`cisco123` is a pretty common password used in Cisco documentation and training labs. It's fine to leave it as-is. It's already in the git history anyways
```suggestion
                        password: cisco123
```